browser/scripts/preset-manager.js

```diff
@@ -121,6 +153,14 @@ PresetManager.prototype.openPreset = function(name) {
 	})
 }
 
+PresetManager.prototype.addObject3d = function(category, title, path) {
```

Comment:
for now this is fine but in future a tag-based system might be more scalable, where object3ds (and other types) are added like other presets but with a 'object3d' tag and the list searches could be filtered to return only objects with certain tags.